packages/loader/container-loader/src/container.ts

```diff
@@ -2330,24 +2327,39 @@ export class Container
 			(message) => this.submitSignal(message),
 			(error?: ICriticalContainerError) => this.dispose(error),
 			(error?: ICriticalContainerError) => this.close(error),
-			Container.version,
-			(dirty: boolean) => this.updateDirtyContainerState(dirty),
+			this.updateDirtyContainerState,
+			this.getAbsoluteUrl,
+			() => this.resolvedUrl?.id,
```

Comment:
Getters are tough to pass - I didn't find a nicer way to eliminate the anonymous wrapping function here unfortunately.  We could consider interface changes to make these full methods (and then pass references similar to updateDirtyContainerState and getAbsoluteUrl above), but there are probably deeper questions to answer about what's appropriate to pass before worrying too much about the syntax I think.

On the flip side, we do get shorter stacks for the downward API calls (since Container can talk to the runtime without passing through the context).